packages/mui-lab/src/internal/pickers/PickersPopper.tsx

```diff
@@ -262,7 +291,16 @@ const PickersPopper = (props: PickerPopperProps) => {
               onTouchStart={onPaperTouchStart}
               ownerState={{ ...ownerState, placement }}
             >
-              {children}
+              <React.Fragment>
+                {children}
+                {showTodayButton && (
+                  <PickersPopperActions ownerState={ownerState}>
+                    <Button data-mui-test="today-action-button" onClick={onSetToday}>
```

Comment:
For a button we don't need `data-mui-test`. For testing, `getByRole('button', { name: todayText })` should do.

Reply:
Sure, I will remove it.

Reply:
@chetas411 Hey, just wanted to ask if you're still working on it? There wasn't any commit in the past 12 days pushed to this branch.

Reply:
@eps1lon Sorry actually my exams are going on so  was not able to work on it. If anyone else wanna continue on it I have no problems in that

Reply:
@chetas411 Thanks for the heads-up and good speed 👍🏻